src/utils/wasm/wasm_exec.d.ts

```
@@ -0,0 +1,10 @@
+import { GoFuncs } from './goFuncs';
+
+declare global {
+  export interface Window extends GoFuncs {
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    Go: any;
+  }
+}
+
+export {};
```

Biome

[error] This empty export is useless because there's another export or import.

This import makes useless the empty export.

Safe fix: Remove this useless empty export.


(lint/complexity/noUselessEmptyExport)